src/libs/db/schema.ts

```diff
@@ -66,3 +67,14 @@ export const verificationTokens = pgTable(
         },
     ],
 );
+
+export const profiles = pgTable("profile", {
+    profileID: text("profile_ID").notNull().primaryKey(),
+    userID: text("user_ID").notNull(),
+    firstName: text("first_name").notNull(),
+    lastName: text("last_name").notNull(),
+    email: text("email").notNull(),
+    universityID: text("studentID").notNull(),
+    gender: genderEnum("gender").notNull(),
+    createdAt: timestamp().defaultNow().notNull(),
```

Comment:
for naming convention we think that we should keep everything be consistent and only use camelCasing for the column in typescript and snake_casing for column in table

also pls keep the table naming consistent,
`universityID: text("studentID").notNull(),`
`universityId: text("university_id").notNull(),`

don't worry about this though because we think we'll not have it lol, see comment below.

```suggestion
    userId: text("user_id").notNull(),
    firstName: text("first_name").notNull(),
    lastName: text("last_name").notNull(),
    email: text("email").notNull(),
    gender: genderEnum("gender").notNull(),
    createdAt: timestamp().defaultNow().notNull(),
```